package.json

```diff
@@ -39,6 +39,7 @@
     "sync-request": "3.0.1",
     "typescript": "1.8.7",
     "validator": "3.33.0",
-    "node-uuid": "1.4.6"
+    "node-uuid": "1.4.6",
+    "nock": "9.0.2"
```

Comment:
... In addition, the solution proposed here is not that far off (and we could even take as the short term.  What we should do per Eric's comment is to install dev dependencies (like build, make file, it's dependencies etc...) globally, test dev dependencies are a separate npm install at the root of the test folder and of course, as done now, true task level dependencies (task runtime) are scoped to the task.